apps/wakunode2/wakunode2.nim

```diff
@@ -269,10 +269,12 @@ proc initNode(conf: WakuNodeConf,
                     else:
                       @[]
 
-    wakuFlags = initWakuFlags(conf.lightpush,
-                              conf.filter,
-                              conf.store,
-                              conf.relay)
+    wakuFlags = CapabilitiesBitfield.init(
```

Comment:
Now that we are refactoring this, would be good to abstract this away. I mean, wakunode should have no knowledge about wakuFlags since it's too low-level and specific to discv5.

I would suggest doing:

```
WakuDiscoveryV5.new(..., capabilities=[Store, Lightpush])
```

which is less likely to mess up.

fine if you want to address in a followup PR

Reply:
Sure, you will see how we will do the ENR extension in the next PR.